.github/scale-config.yml

```diff
@@ -27,6 +27,11 @@ runner_types:
     os: linux
     max_available: 50
     disk_size: 150
+  linux.16xlarge.nvidia.gpu:
+    instance_type: g3.16xlarge
+    os: linux
+    max_available: 25
```

Comment:
We should probably make this lower considering this instance type is potentially very expensive  ($3.11/hr)

How about setting a max_available of only 10?

Reply:
@seemethere makes sense! done